module/modmanager/manager.go

```diff
@@ -581,10 +593,20 @@ func (m *module) startProcess(
 	if err := modlib.CheckSocketAddressLength(m.addr); err != nil {
 		return err
 	}
+
+	// Start module process with supplied log level or "debug" if none is
+	// supplied and module manager has a DebugLevel logger.
+	args := []string{m.addr}
+	if m.logLevel != "" {
+		args = append(args, fmt.Sprintf(logLevelArgumentTemplate, m.logLevel))
+	} else if logger.Level().Enabled(zapcore.DebugLevel) {
+		args = append(args, fmt.Sprintf(logLevelArgumentTemplate, "debug"))
+	}
+
```

Comment:
[nit/opt] Super minor. This could be AFTER the pconf init below, and thus not need the extra args variable here (just modify pconf.Args instead.)